keras/backend/mxnet_backend.py

```diff
@@ -1202,13 +1202,18 @@ def gather(reference, indices):
 
 
 @keras_mxnet_symbol
-def embedding(data, weight, input_dim, output_dim):
+def embedding(data, weight, input_dim, output_dim, sparse_grad=False):
```

Comment:
K.embedding() is called in keras.layers.Embedding class for mxnet backend. Please also update the function signature there, and test in a end to end example that use Embedding layer. (e.g. examples using imdb dataset)

Reply:
We are passing default value as `False` for `sparse_grad` in the API signature so making a change there is not necessary